Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py

```diff
@@ -8,8 +8,29 @@
 QUOTE_MARKERS = ['<div class="gmail_quote">',
                  '<hr tabindex="-1" style="display:inline-block; width:98%"><div id="divRplyFwdMsg"']
 
+
+def get_query_window():
+    """
+    Check if the user defined the list `EmailCommunicationQueryWindow` to give a custom value for the time to query back
+    for related incidents. If yes, use this value, else use the default value of 60 days.
+    """
+    user_defined_time = demisto.executeCommand('getList', {'listName': 'EmailCommunicationQueryWindow'})
+    if is_error(user_defined_time):
+        demisto.debug(f'The following error occurred while trying to load the EmailCommunicationQueryWindow list - '
+                      f'{user_defined_time[0].get("Contents")}.\nUsing the default query time - 60 days')
```

Comment:
if you got an error here, you may not have an array in `user_defined_time`. and there for have an exception when doing `user_defined_time[0]`